features/inventory/validating_product_quantity_on_checkout_complete.feature

```diff
@@ -0,0 +1,35 @@
+@inventory
+Feature: Validating product quantity on checkout complete
+    In order to be sure that products I buy are available
+    As a Customer
+    I want to have them validated on checkout complete
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "Iron Maiden T-Shirt" priced at "€12.54"
+        And this product is tracked by the inventory
+        And there are 5 items of product "Iron Maiden T-Shirt" available in the inventory
```

Comment:
Shouldn't we talk about `units` instead of `items` when it's about the inventory?